Dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM eclipse-temurin:21.0.7_6-jre-noble
 
 ARG BUILD_DATE
+ARG BUILDPLATFORM
```

Comment:
Don't we need to use `TARGETPLATFORM`? It was my understanding that BUILDPLATFORM was the build machine's platform and `TARGETPLATFORM` is the platform we are building for.